system/Database/MySQLi/Connection.php

```diff
@@ -488,7 +494,7 @@ public function _indexData(string $table)
 				}
 				else
 				{
-					throw new \LogicException('parsing key string failed.');
+					throw new \LogicException('Parsing key string failed.');
```

Comment:
If you wouldn't mind moving this one to the Database lang file too, that would be great!